public/stylesheets/style.less

```diff
@@ -42,14 +42,15 @@ div.card {
     display: flex;
     flex-direction: column;
     justify-content: center;
+    width: 100%;
 
     h2 {
       margin: 0;
       padding-bottom: 10px;
     }
 
     a {
-      width: 100%;
```

Comment:
Is it necessary to remove the width?

Reply:
Yes it is necessary because the user could click on a whitespace and would trigger the change on the page